mappings/net/minecraft/entity/EntityLike.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_5568 net/minecraft/entity/EntityLike
```

Comment:
```suggestion
CLASS net/minecraft/class_5568 net/minecraft/entity/EntityAccess
```
Can be another solution

Reply:
Per our world naming guide we'd name it like `StoredEntity` `PersistentEntity` etc